src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialServiceSupport.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import java.text.SimpleDateFormat
+import java.util
+import java.util.Date
+
+import com.google.api.services.sheets.v4.model.{SpreadsheetProperties, ValueRange}
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.dataaccess.{ThurloeDAO, TrialDAO}
+import org.broadinstitute.dsde.firecloud.model.Trial.{TrialProject, UserTrialStatus}
+import org.broadinstitute.dsde.firecloud.model.{UserInfo, WorkbenchUserInfo}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait TrialServiceSupport extends LazyLogging {
+
+  private val dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
+
+  def makeSpreadsheetProperties(title: String): SpreadsheetProperties = {
+    val dateString = dateFormat.format(new Date())
+    new SpreadsheetProperties().setTitle(s"$title: $dateString")
+  }
+
+  // TODO: Look further into the Onix "User Agreement" field
+  def makeSpreadsheetValues(managerInfo: UserInfo, trialDAO: TrialDAO, thurloeDAO: ThurloeDAO, majorDimension: String, range: String)
+    (implicit executionContext: ExecutionContext): Future[ValueRange] = {
+    import scala.collection.JavaConverters._
+    val claimedProjects: Seq[TrialProject] = trialDAO.projectReport
+    val availableProjects: Seq[TrialProject] = trialDAO.availableProjectReport
+    val trialStatuses = Future.sequence(claimedProjects.map { p =>
+      if (p.user.isDefined)
+        thurloeDAO.getTrialStatus(p.user.get.userSubjectId, managerInfo)
+      else
+        Future[Option[UserTrialStatus]](None)
+    })
+    trialStatuses.map { userTrialStatuses =>
+      val headers = List("Project Name", "User Subject Id", "User Email", "Enrollment Date", "User Agreement").map(_.asInstanceOf[AnyRef]).asJava
```

Comment:
I realize this isn't in the requirements, but now that I'm looking at the code: it probably makes sense to add termination date here too.